packages/accounts-2fa/2fa-server.js

```diff
@@ -29,7 +29,13 @@ Accounts._is2faEnabledForUser = selector => {
     }
   }
 
-  const user = Meteor.users.findOne(selector) || {};
+  const user = Meteor.users.findOne(selector);
+  if (!user) {
+    throw new Meteor.Error(
+      500,
```

Comment:
Maybe 404 error would be more appropriate here?

Reply:
I think it makes more sense. The docs also need to be updated to 404 in that case.

Reply:
Sorry to barge in, but why use HTTP error codes?? This is DDP, won't a descriptive title be better? 
https://github.com/meteor/meteor/blob/365604c91020db2c36b491e7ea332ed730492994/packages/ddp-server/livedata_server.js#L663

Reply:
It's the pattern that the packages are using, but sure enough, I'm going to use a descriptive code in this case.